heritagefund/funding-frontend
# Only send mission to SF when present for org.  Doesnt send null as be…

## Proposed changes

Only send mission to SF when present for org.  Doesn't send null - as before - which overwrites existing SF values.  Requires PortApplication.cls v1.1

## Further comments

Tests:
- Send PEF with no mission - confirm Org in SF
- Manually update org in SF, so that mission exists
- Send 3-10K from FFE and confirm that SF version of org has manually entered mission preserved.
- Update mission in FFE. Send a second 3-10K from FFE to SF.  Confirm Mission updated in SF with FFE submission.
